openresty/crowdsec_openresty.conf

```diff
@@ -1,14 +1,20 @@
 lua_package_path '$prefix/../lualib/plugins/crowdsec/?.lua;;';
 lua_shared_dict crowdsec_cache 50m;
+resolver local=on ipv6=off;
```

Comment:
For this, I think we don't want to add by default a resolver in the default config file. Because in some cases users already configured a resolver in higher openresty configuration for all vhosts.
We already have documentation on it, to enable only when you use captcha feature.

Reply:
Ok Removed